cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -43,6 +44,7 @@ const tagsButton = Button({ id: 'clickable-show-tags' });
 const tagsAccordionButton = instancesTagsSection.find(Button('Tags'));
 const emptyResultsMessage = 'Choose a filter or enter a search query to show results.';
 const browseButton = Button({ id: 'mode-navigation-browse' });
+const section = Section({ id: 'browse-inventory-results-pane' });
```

Comment:
'section' is assigned a value but never used. Allowed unused vars must match /React/u

Reply:
Fixed